src/app/public/modules/shared/initialization.service.spec.ts

```diff
@@ -19,92 +19,123 @@ import {
   HelpWidgetConfig
 } from './widget-config';
 
-const NO_OP_FUNC: Function = () => {
-};
+import {
+  ConfigExtension
+} from './config.extension';
+
+import {
+  of
+} from 'rxjs';
+
+import {
+  fakeAsync,
+  tick
+} from '@angular/core/testing';
 
 describe('HelpInitializationService', () => {
 
   beforeEach(() => {
-    spyOn(BBHelpClient, 'load').and.callFake(NO_OP_FUNC);
+    spyOn(BBHelpClient, 'load').and.returnValue(Promise.resolve());
```

Comment:
the lack of typing on `BBHelpClient` i think was confusing previous devs. it returns a promise. so this shouldve always been returning a promise. this is also partially why the subsequent tests were tweaked to include `fakeAsync`/`tick`